app/src/main/java/org/android/go/sopt/Extensions.kt

```diff
@@ -0,0 +1,15 @@
+package org.android.go.sopt
+
+import android.content.Context
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
+import org.android.go.sopt.databinding.ActivitySignUpBinding
+
+fun ActivitySignUpBinding.idSucceed() = (this.etId.text?.length ?: 0) in 6..10
+fun ActivitySignUpBinding.pwSucceed() = (this.etPw.text?.length ?: 0) in 8..12
+fun EditText.get() = this.text.toString()
+fun TextView.plus(value : String) {
```

Comment:
```suggestion
operator fun TextView.plus(value : String) {
```

이렇게 적용하면

```kotlin
binding.textView + ""
```

이렇게 써도 되누ㅋㅋㅋㅋ